plone/app/portlets/portlets/navigation.py

```diff
@@ -217,11 +217,11 @@ def root_item_class(self):
             return 'navTreeCurrentItem'
         return ''
 
-    def root_icon(self):
-        plone_layout = getMultiAdapter((self.context, self.request),
-            name='plone_layout')
-        icon = plone_layout.getIcon(self.getNavRoot())
-        return icon
+#     def root_icon(self):
+#         plone_layout = getMultiAdapter((self.context, self.request),
+#             name='plone_layout')
+#         icon = plone_layout.getIcon(self.getNavRoot())
+#         return icon
```

Comment:
ok - thanks for the hint

I reverted that (not sure, ... might break some tests or anything else). Should be cleaned up asap though.